File: Easy/Eg1: 1dArrayRunningSum/main.js
var runningSum = function(nums) {
    
    for (i=1;i<nums.length;i++) {
          nums[i]=nums[i-1]+nums[i]
    }
    return nums
};

nums = [1,2,3,4]
console.log(`The running sum of ${nums} is: ${runningSum(nums)}`)

File: Easy/Eg1: 1dArrayRunningSum/main.py
def runningSum(nums):
    y=[]
    x_index=0
    for x in nums:
        if x_index != 0:
          y.append(sum(nums[:x_index] + [nums[x_index]]))
        else:
            y.append(x)
        x_index+=1
    return y

nums = [1,2,3,4]
print(f"The running sum of {nums} is: {runningSum(nums)}")